Unix installation skips `src`

Current Unix installation method would through an error kinda like below
```
deptools/gx-go_v1.8.0 rewrite
go build -ldflags "-X ipfscluster.Commit=2d072d8b53fc5846026c91d39620e0f3d57b3450"
cluster.go:11:2: cannot find package "github.com/ipfs/ipfs-cluster/adder" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/adder (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/adder (from $GOPATH)
cluster.go:12:2: cannot find package "github.com/ipfs/ipfs-cluster/adder/local" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/adder/local (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/adder/local (from $GOPATH)
cluster.go:13:2: cannot find package "github.com/ipfs/ipfs-cluster/adder/sharding" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/adder/sharding (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/adder/sharding (from $GOPATH)
allocate.go:10:2: cannot find package "github.com/ipfs/ipfs-cluster/api" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/api (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/api (from $GOPATH)
cluster_config.go:14:2: cannot find package "github.com/ipfs/ipfs-cluster/config" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/config (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/config (from $GOPATH)
cluster.go:15:2: cannot find package "github.com/ipfs/ipfs-cluster/pstoremgr" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/pstoremgr (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/pstoremgr (from $GOPATH)
cluster.go:16:2: cannot find package "github.com/ipfs/ipfs-cluster/rpcutil" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/rpcutil (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/rpcutil (from $GOPATH)
cluster.go:17:2: cannot find package "github.com/ipfs/ipfs-cluster/state" in any of:
	/usr/local/go/src/github.com/ipfs/ipfs-cluster/state (from $GOROOT)
	/home/kishansagathiya/go/ipfs/src/github.com/ipfs/ipfs-cluster/state (from $GOPATH)
make: *** [Makefile:32: build] Error 1
```
I think this is because current Unix setup has skip `src` and directly created `github.com/ipfs/ipfs-cluster` in $GOPATH

--- a/content/documentation/download.md
+++ b/content/documentation/download.md
@@ -52,8 +52,8 @@ The following requirements apply to the installation from source:
 In order to build and install IPFS Cluster in Unix systems follow the steps:
 
 ```
-git clone https://github.com/ipfs/ipfs-cluster.git $GOPATH/github.com/ipfs/ipfs-cluster
-cd $GOPATH/github.com/ipfs/ipfs-cluster
+git clone https://github.com/ipfs/ipfs-cluster.git $GOPATH/src/github.com/ipfs/ipfs-cluster
+cd $GOPATH/src/github.com/ipfs/ipfs-cluster
 make install
 ```
 

